src/xclim/indicators/land/_snow.py

```diff
@@ -254,3 +256,26 @@ class SnowWithIndexing(ResamplingIndicatorWithIndexing):
     abstract="Number of days when the snow amount is greater than or equal to a given threshold.",
     compute=xci.snw_days_above,
 )
+
+holiday_snow_days = Snow(
+    title="Christmas snow days",
+    identifier="holiday_snow_days",
+    units="1",
```

Comment:
```suggestion
    units="days",
```
The output is a number of days, shouldn't units be "days" ?

Reply:
Right. Forgot to change this. Thanks!